devices/YarpRobotLoggerDevice/app/robots/ergoCubGazeboV1/yarp-robot-logger.xml

```diff
@@ -0,0 +1,83 @@
+<!-- Copyright (C) 2019-2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+This software may be modified and distributed under the terms of the
+BSD-3-Clause license. -->
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<device  xmlns:xi="http://www.w3.org/2001/XInclude" name="yarp-robot-logger" type="YarpRobotLoggerDevice">
+  <param name="robot">ergocubSim</param>
+  <param name="sampling_period_in_s">0.002</param>
+  <param name="rgb_cameras_fps">(30)</param>
+  <param name="video_codec_code">mp4v</param>
+  <param name="port_prefix">/yarp-robot-logger</param>
+  <param name="text_logging_subnames">("ergoCubGazeboV1/yarprobotinterface")</param>
+
+  <group name="Telemetry">
+    <param name="save_period">600.0</param>
+  </group>
+
+  <group name="ExogenousSignals">
+    <param name="vectors_collection_exogenous_inputs">("RobotDynamicsEstimator", "FixedBaseTSID")</param>
+    <param name="vectors_exogenous_inputs">()</param>
+
+    <group name="Walking">
+      <param name="local">"/yarp-robot-logger/exogenous_signals/walking:i"</param>
+      <param name="remote">"/walking-coordinator/logger/data:o"</param>
+      <param name="signal_name">"walking"</param>
+      <param name="carrier">"udp"</param>
+    </group>
+
+    <group name="RobotDynamicsEstimator">
+      <param name="local">"/yarp-robot-logger/exogenous_signals/rde:i"</param>
+      <param name="remote">"/robot-dynamics-estimator/data:o"</param>
+      <param name="signal_name">"rde"</param>
+      <param name="carrier">"udp"</param>
+    </group>
+
+    <group name="FixedBaseTSID">
+      <param name="local">"/yarp-robot-logger/exogenous_signals/FixedBaseTSID:i"</param>
+      <param name="remote">"/FixedBaseTSID"</param>
+      <param name="signal_name">"FixedBaseTSID"</param>
+      <param name="carrier">"udp"</param>
+    </group>
+  </group>
```

Comment:
Probably you may comment the groups and remove them from the list

Reply:
Done